lib/auth/apiserver.go

```diff
@@ -75,9 +75,8 @@ func NewAPIServer(config *APIConfig) http.Handler {
 	srv.GET("/:version/authorities/:type/:domain", srv.withAuth(srv.getCertAuthority))
 	srv.GET("/:version/authorities/:type", srv.withAuth(srv.getCertAuthorities))
 
-	// Generating certificates for user and host authorities
+	// Generating certificates for host authorities
 	srv.POST("/:version/ca/host/certs", srv.withAuth(srv.generateHostCert))
-	srv.POST("/:version/ca/user/certs", srv.withAuth(srv.generateUserCert))
```

Comment:
I think you should keep this method for now in 4.1 with a DELTE in marker, otherwise `tctl` of 4.0 version will stop working.

Reply:
Fixed.